cli/web_worker.rs

```diff
@@ -158,7 +158,9 @@ impl WebWorker {
           &mut js_runtime,
           Some(inspector_server.clone()),
         ))
-      } else if program_state.flags.coverage || program_state.flags.repl {
+      } else if program_state.flags.coverage_dir.is_some()
+        || program_state.flags.repl
+      {
```

Comment:
This will conflict with #8659

Reply:
> @caspervonb why is this PR breaking?

Only changes unstable, but because CLI usage changes from

```
$ deno test --unstable --coverage 
```

to 
```
$ deno test --unstable --coverage path_to_directory
```

Reply:
> Also is this env var unset after coverage is done?

We're setting in in the current process so on exit it should be back to normal. Env inheritance only goes downwards not upwards.

> This will conflict with #8659

Hm yeah but trivially so?

Reply:
Merged, no problem 👍